libraries/helpers_windows.rb

```diff
@@ -66,9 +71,9 @@ def build_rule(new_resource)
         if new_resource.direction.to_sym == :out
           parameters['localip'] = new_resource.source ? fixup_cidr(new_resource.source) : 'any'
           parameters['localport'] = new_resource.source_port ? port_to_s(new_resource.source_port) : 'any'
-          parameters['interfacetype'] = new_resource.source_interface ? new_resource.source_interface : 'any'
+          parameters['interfacetype'] = new_resource.interface ? new_resource.interface : 'any'
           parameters['remoteip'] = new_resource.destination ? fixup_cidr(new_resource.destination) : 'any'
-          parameters['remoteport'] = port_to_s(new_resource.dest_port) ? new_resource.dest_port : 'any'
+          parameters['remoteport'] = new_resource.dest_port ? port_to_s(new_resource.dest_port) : 'any'
```

Comment:
In debugging a windows firewall recipe run, it turned out that it was a kind of typo. Should be like that to make it work according to syntax rules of advfirewall command on Windows